test/src/test/java/hudson/tasks/ShellTest.java

```diff
@@ -190,10 +191,10 @@ public void unixUnstableCodeZeroIsSameAsUnset() throws Exception {
     @Issue("JENKINS-40894")
     @Test
     @LocalData
-    public void canLoadUnstableReturnFromDisk() throws Exception {
+    public void canLoadUnstableReturnFromDisk() {
         FreeStyleProject p = (FreeStyleProject) rule.jenkins.getItemByFullName("test");
         Shell shell = (Shell) p.getBuildersList().get(0);
-        assertEquals("unstable return", Integer.valueOf(1), shell.getUnstableReturn());
+        assertEquals("unstable return", (Integer) 1, shell.getUnstableReturn());
```

Comment:
FTR This and the similar ones are to prevent "Ambiguous method call", if we were using just `1`.